api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -1104,9 +1103,12 @@ message LoadBalancingPolicy {
     reserved "config";
 
     // Required. The name of the LB policy.
-    string name = 1;
+    string name = 1 [deprecated = true, (envoy.annotations.deprecated_at_minor_version) = "3.0"];
 
-    google.protobuf.Any typed_config = 3;
+    google.protobuf.Any typed_config = 3
+        [deprecated = true, (envoy.annotations.deprecated_at_minor_version) = "3.0"];
```

Comment:
I thought we can delete them instead of deprecating since the whole message was `not-implemented-hide` before, and the implementation doesn't support this either.

Reply:
I'd be okay with fully removing these fields too, unless that causes problems.  But I think there's very little cost to leaving them here and marking them deprecated.

Reply:
Hmm. I tried to remove the whole Policy message from v3 only earlier and it failed the presubmit because it was expecting some v3 message to correspond to the v2 version of the same message. (That's why I ended up deprecating v2 fields in an earlier commit, which I've since undone.)

Removing these fields seems to be fine, though, so I've done that.

Reply:
Looks good to me!

Reply:
Thanks. While there's little cost to leaving them here, but then the doc will be confusing as we never use those field in implementation.